### DISCUSSION TITLE
Hacking help by copilot

### DISCUSSION THREAD

**Discussion by milosz0542** [community/community#67333] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Should copilot write keyloggers and others hacking scripts? For example chatGPT banned this function.
![image](https://github.com/community/community/assets/10118724/dcf2ff47-9969-45ba-906e-57018524ea6d)

**Comment by mkare** [comment, marked as answer]
No, Copilot should not write keyloggers and other hacking scripts. Keyloggers are malicious programs that record all keystrokes typed on a computer, including passwords and other sensitive information. Hacking scripts can be used to gain unauthorized access to computer systems or networks, or to steal data.

Copilot is a powerful tool that can help developers write code more quickly and efficiently. However, it is important to use Copilot responsibly. Copilot should not be used to generate code that could be used for malicious purposes.

ChatGPT has banned the generation of keyloggers and other hacking scripts because it is important to use AI responsibly. ChatGPT does not want to be used to create tools that could be used to harm others.

**Reply by JT348** [reply]
no i want to hack nitrotype and it wont let me
